src/entrypoint/graphql-core-server.ts

```diff
@@ -150,10 +150,11 @@ const resolvers = {
     },
     getLevels: () => levels,
     getLimits: (_, __, { user }) => {
+      const transactionLimits = selectTransactionLimits({ level: user.level })
       return {
-        oldEnoughForWithdrawal: yamlConfig.limits.oldEnoughForWithdrawal,
-        withdrawal: yamlConfig.limits.withdrawal.level[user.level],
-        onUs: yamlConfig.limits.onUs.level[user.level],
+        oldEnoughForWithdrawal: transactionLimits.oldEnoughForWithdrawalLimit,
```

Comment:
Ideally I'd like to secure this change with a test, but I'm not sure if now is the right time or what the best place would be with the other tests work you all were doing recently? (cc @dolcalmi)

Reply:
I think you can add extra tests related to this here `test/integration/02-user-wallet/02-index.spec.ts`

Reply:
also if you want to test graphql query then add it here `test/integration/entrypoint/graphql.spec.ts`

Reply:
Removing this planned test from the scope since we have planned broader graphql testing work